dowhy/gcm/independence_test/__init__.py

```diff
@@ -1 +1,28 @@
 from .kernel import kernel_based, approx_kernel_based
+
+
+def independence_test(X, Y, conditioned_on=None, method="kernel"):
+    """Performs a (conditional) independence test.
+    Three methods for (conditional) independence test are supported at the moment:
```

Comment:
from a user's perspective, there are only 2 methods, right (and three papers)? The approx-kernel will also include conditional and unconditional.